packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -256,8 +261,9 @@ class Editor extends React.Component {
         font-size: ${theme.typography.body1.fontSize};
 
         /* 80px - 2 * 1px borders  */
+        max-height: ${maxHeight || "auto"};
```

Comment:
I think instead of `|| "auto"` here, we should just make the `maxHeight` props have a default value of "auto".